configurators/pom.xml

```diff
@@ -124,21 +125,6 @@
             <artifactId>jdom</artifactId>
             <version>1.1</version>
         </dependency>
-        <dependency>
```

Comment:
Removing jaxen makes several unit tests fail

Reply:
Fixed by using jdom 2 and jaxen 1.2.0